Source/SIMPLib/Filtering/ComparisonSet.h

```diff
@@ -53,6 +54,7 @@ class SIMPLib_EXPORT ComparisonSet : public AbstractComparison
   PYB11_PROPERTY(int UnionOperator READ getUnionOperator WRITE setUnionOperator)
   PYB11_METHOD(void addComparison ARGS AbstractComparisonPtr,comparison)
   PYB11_METHOD(void insertComparison ARGS int,index AbstractComparisonPtr,comparison)
```

Comment:
I am going to assume spaces are important here, too.  Did I miss this in the earlier commits, or is not having a space actually valid here?